SafeguardSessions/README.md

```diff
@@ -43,6 +43,10 @@ To use the connector,
 
 You can read a more comprehensive guide on using the Power BI Connector and the Report Template in the One Idenity Safeguard Power BI Connector Tutorial that can be found on the [Technical documents for One Identity Safeguard for Priviledged Sessions] page.
 
+### Fetching data from multiple SPS instances
+
+You can fetch data from multiple SPS instances by repeating steps 10. to 13. from [Installation and usage]. If you want to append your queries, follow the [Append queries] official tutorial.
```

Comment:
For sake of clarity, maybe, if you think this helps.

#Optional: "using different SPS address and query parameters"

Reply:
1. IP: I don't think it adds much value. I'll let the other decide.
2. Query parameters: The readme does not mention the query parameters, only the standalone guide does.